index.ts

```diff
@@ -260,7 +260,7 @@ export async function getLinkPreview(
     throw new Error(`link-preview-js did not receive a valid url or text`);
   }
 
-  const detectedUrl = text
+  const detectedUrl = (options?.proxyUrl ? options.proxyUrl.concat(text) : text)
```

Comment:
can you explain how is this supposed to work? the entire text gets concatenated at the end of the proxyUrl, but then it gets spliced? 

Can you also add a test for this?

Reply:
oh, i think i misunderstood how detectedUrl works. the idea was to concat the proxyUrl + url for the fetch request. i just need to move the logic, i think.

Reply:
moved the logic, need to think about a good test